pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -3314,6 +3314,27 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 			})
 		})
 	})
+	Context("with downwardmetrics virtio serial", func() {
+		It("should accept a single virtio serial", func() {
+			enableFeatureGate(virtconfig.DownwardMetricsFeatureGate)
+			vmi := api.NewMinimalVMI("testvmi")
+
+			vmi.Spec.Domain.Devices.DownwardMetrics = &v1.DownwardMetrics{}
+
+			causes := validateDownwardMetrics(k8sfield.NewPath("fake"), &vmi.Spec, config)
```

Comment:
supernit - I'd inline this

`Expect(validateDownwardMetrics(k8sfield.NewPath("fake"), &vmi.Spec, config)).To(BeEmpty())`

Reply:
Done.